io/io/inc/TFile.h

```diff
@@ -174,6 +174,7 @@ class TFile : public TDirectoryFile {
 public:
    /// TFile status bits. BIT(13) is taken up by TObject
    enum EStatusBits {
+      kReproducible  = BIT(9),
```

Comment:
```BIT(9)``` is used in TDirectoryFile  [and TDirectoryFile needs an update to rename its bits enum to EStatusBits]

Reply:
I changed it to BIT(17) and also add EStatusBits for enum in TDirectoryFile